docs/snapshots.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Share Errors with Snapshots
+layout: docs
+---
+
+Thanks for trying out the snapshots beta! This page explains what snapshots are and how to manage them. There's also a FAQ section at the end.
```

Comment:
if you wanna get fancy, link the text about the faq down to the FAQ below?

Reply:
I would love to but I can't figure out how to make it an anchor link =\

Reply:
fwiw, jekyll will let you write arbitrary html in the middle of markdown